### DISCUSSION TITLE
What's the difference between OnClick and @onclick for FluentButton?

### DISCUSSION THREAD

**Discussion by MarvinKlein1508** [microsoft/fluentui-blazor#2472] [Q&A]
The component offers both and both work jsut fine for me. So what's the purpose of the custom EventCallback OnClick?

**Comment by dvoituron** [comment]
There is no difference: 
- `@onclick` is the default attribute proposed by Blazor.
- `OnClick` is present to respect the same syntax as the other attributes of our components (default and custom events)
